README.md

```diff
@@ -134,6 +134,37 @@ that handle styles based on values defined in a theme.
 <Box bg='tomato' />
 ```
 
+### Use with Object Styles
+
+*Styled Components*
+
+```jsx
+import styled from 'styled-components';
+import { color } from 'styled-system';
+
+const Text = styled.span(props => (
+  {
+    color: color(props),
```

Comment:
There might be a better way to do this in SC. I was specifically solving for the Emotion use case.

Reply:
Both styled-components and emotion can accept multiple arguments, so the example you have below for emotion probably makes sense to use as a generic one – I don't think there's a need for both examples

Reply:
Cool. I have some time today to update this. Thanks for the feedback!